packages/flutter_tools/lib/src/isolated/resident_web_runner.dart

```diff
@@ -605,6 +606,7 @@ class ResidentWebRunner extends ResidentRunner {
           kNoneWorker,
           true,
           debuggingOptions.nativeNullAssertions,
+          kBaseHref
```

Comment:
nit: trailing comma:
```suggestion
          kBaseHref,
```